hedyorg/hedy
# [RU] Fill content student details

Trello issue: https://trello.com/c/kRyVrNrS/15-feature-fill-content-of-student-details-even-just-images-from-the-prototype-will-suffice-high-priority-easy

**Description**
Fills the content of the "Student Details" for now just using images, these need to replaced later with actual graphs/data/error logs. It adds the correct student name and level, which should correspond to the student clicked on. 

**Relevant technical choices**
* I've moved all images from our project into a `new_media_lab` folder within the images folder. They can always be moved into the parent folder at a later stage, but now it's easier to keep track of them (as many of them are placeholders and will need to be removed in the future).
* In `statistic.py` I've passed the data of the individual student that was clicked, as we need to show it on the card. Good to note that currently only one student details card can be opened at the same time, i.e. if you click a second student, it replaces the card instead of opening a new one. I don't think it is a priority to change this now, but maybe good to consider :).

***Test instructions*
* Start a local environment, login to a teacher account and checkout this branch.
* Navigate to `for_teachers` > `CLASS1`  > `class_live`.
* You should see our live statistics dashboard.
* In the `Student List`, click on one of the students.
* A `Student Details` card should open. It should look like the screenshot below 
  ![image](https://user-images.githubusercontent.com/43582255/230992664-60a4b83c-8212-40b6-9eab-e95103dacd49.png)
* Verify the `Name` and `Current level` are correct, corresponding to the previously clicked student.
*  Verify the `Student List` card still shows all students.
* Close the `Student details` card, and see it disappear.
* Click on another student name, and again verify the card opens, shows the correct name and level and content. 
* Click on another student name (without closing the card first!), see that there is only one card containing the data of the last clicked student.
